drivers/pds/lib/utils.go

```diff
@@ -1441,58 +1506,69 @@ func CreateTpccWorkloads(dataServiceName string, deploymentID string, scalefacto
-		dep, err = CreatepostgresqlWorkload(dnsEndpoint, pdsPassword, scalefactor, iterations, deploymentName, namespace)
+		dep, err = CreatepostgresqlWorkload(dnsEndpoint, pdsPassword, params.ScaleFactor, params.Iterations, params.DeploymentName, params.Namespace)
 		if err != nil {
-			log.Errorf("An Error Occured while creating postgresql workload %v", err)
-			return nil, nil, err
+			return nil, nil, fmt.Errorf("error occured whilecreating postgresql workload %v", err)
 		}
 
 	case rabbitmq:
 		env := []string{"AMQP_HOST", "PDS_USER", "PDS_PASS"}
 		command := "while true; do java -jar perf-test.jar --uri amqp://${PDS_USER}:${PDS_PASS}@${AMQP_HOST} -jb -s 10240 -z 100 --variable-rate 100:30 --producers 10 --consumers 50; done"
-		pod, err = CreateRmqWorkload(dnsEndpoint, pdsPassword, namespace, env, command)
+		pod, err = CreateRmqWorkload(dnsEndpoint, pdsPassword, params.Namespace, env, command)
 		if err != nil {
-			log.Errorf("An Error Occured while creating rabbitmq workload %v", err)
-			return nil, nil, err
+			return nil, nil, fmt.Errorf("error occured while creating rabbitmq workload %v ", err)
 		}
 
 	case redis:
 		env := []string{"REDIS_HOST", "PDS_USER", "PDS_PASS"}
 		command := "redis-benchmark -a ${PDS_PASS} -h ${REDIS_HOST} -r 10000 -c 1000 -l -q --cluster --user ${PDS_USER}"
-		pod, err = CreateRedisWorkload(deploymentName, redisStressImage, dnsEndpoint, pdsPassword, namespace, env, command)
+		pod, err = CreateRedisWorkload(params.DeploymentName, redisStressImage, dnsEndpoint, pdsPassword, params.Namespace, env, command)
 		if err != nil {
-			log.Errorf("An Error Occured while creating redis workload %v", err)
-			return nil, nil, err
+			return nil, nil, fmt.Errorf("error occured while creating redis workload %v ", err)
 		}
 
 	case cassandra:
-		cassCommand := deploymentName + " write no-warmup n=1000000 cl=ONE -mode user=pds password=" + pdsPassword + " native cql3 -col n=FIXED\\(5\\) size=FIXED\\(64\\)  -pop seq=1..1000000 -node " + dnsEndpoint + " -port native=9042 -rate auto -log file=/tmp/" + deploymentName + ".load.data -schema \"replication(factor=3)\" -errors ignore; cat /tmp/" + deploymentName + ".load.data"
-		dep, err = CreatecassandraWorkload(cassCommand, deploymentName, namespace)
+		cassCommand := fmt.Sprintf("%s write no-warmup n=1000000 cl=ONE -mode user=pds password=%s native cql3 -col n=FIXED\\(5\\) size=FIXED\\(64\\)  -pop seq=1..1000000 -node %s -port native=9042 -rate auto -log file=/tmp/%s.load.data -schema \"replication(factor=3)\" -errors ignore; cat /tmp/%s.load.data", params.DeploymentName, pdsPassword, dnsEndpoint, params.DeploymentName, params.DeploymentName)
+		dep, err = CreateDeploymentWorkloads(cassCommand, params.DeploymentName, cassandraStresImage, params.Namespace)
 		if err != nil {
-			log.Errorf("An Error Occured while creating cassandra workload %v", err)
-			return nil, nil, err
+			return nil, nil, fmt.Errorf("error occured while creating cassandra workload %v ", err)
```

Comment:
nit: please find and remove ALL extra space from messages

Reply:
done